sormas-api/src/main/java/de/symeda/sormas/api/i18n/I18nProperties.java

```diff
@@ -338,14 +338,21 @@ private I18nProperties() {
 	}
 
 	public static String getContinentName(String defaultName) {
-		String name =
-			getInstance(userLanguage.get()).continentProperties.getString("continent." + defaultName.replace(" ", "_").toUpperCase() + ".name");
+		// remove spaces und everything in brackets following (e.g. "Australia (Continent)" => "AUSTRALIA")
+		defaultName = defaultName.substring(0, defaultName.contains("(") ? defaultName.indexOf("(") : defaultName.length())
+			.trim()
+			.replace(" ", "_")
+			.toUpperCase();
+		String name = getInstance(userLanguage.get()).continentProperties.getString("continent." + defaultName + ".name");
 		return name != null ? name : defaultName;
```

Comment:
The behaviour of the method has changed:
`I18nProperties.getContinentName("Some unknown country (Continent)")` => `"SOME_UNKNOWN_COUNTRY"`
(was before: `"Some unknown country (Continent)"`)

Reply:
Optional: add tests to `I18nPropertiesTest`